numerics/poisson_series_body.hpp

```diff
@@ -653,6 +653,33 @@ Value PiecewisePoissonSeries<Value, degree_, Evaluator>::operator()(
   return series_[it - bounds_.cbegin() - 1](t);
 }
 
+template<typename Value,
+         int degree_,
+         template<typename, typename, int>
+         class Evaluator>
+inline std::function<Complexification<Value>(AngularFrequency const&)>
+PiecewisePoissonSeries<Value, degree_, Evaluator>::FourierTransform() const {
+  // TODO(egg): consider pre-evaluating |*this| at all points used by the
+  // Gaussian quadratures, removing the lifetime requirement on |*this| and
+  // potentially speeding up repeated evaluations of the Fourier transform.
+  return [this](AngularFrequency const& ω) {
+    Interval<Instant> const time_domain{t_min(), t_max()};
+    Instant const t0 = time_domain.midpoint();
+    Primitive<Complexification<Value>, Instant> integral;
+    for (int k = 0; k < series_.size(); ++k) {
+      integral += quadrature::GaussLegendre<std::max(1, (degree_ + 1) / 2)>(
+          [&f = series_[k], t0, ω](
+              Instant const t) -> Complexification<Value> {
```

Comment:
Pass `Instant` by reference.  For better or for worse, that's what we do everywhere.